rpc/documentation/api.md

```diff
@@ -627,6 +627,23 @@ but not all sidechain blocks may be known by the wallet.
 
 ___
 
+#### `GetAccountExtendedKey`
+
+The `GetAccountExtendedKey` method queries the wallet for an account pubkey. If no account is provided, then it returns the master pubkey.
+
+**Request:** `GetAccountExtendedKeyRequest`
+
+- `string account`: The account name to retrieve the pubkey.
```

Comment:
the grpc api refers to accounts as a uint32.  This method must remain consistent.